airbyte-cdk/python/airbyte_cdk/sources/declarative/yaml_declarative_source.py

```diff
@@ -65,9 +66,11 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         return [self._factory.create_component(stream_config, config, True)() for stream_config in self._stream_configs()]
 
     def _read_and_parse_yaml_file(self, path_to_yaml_file):
-        with open(path_to_yaml_file, "r") as f:
```

Comment:
open() only works running locally and I am not sure how. this passes SAT, something we should follow up on. But replacing this with how we read spec files fixes the issue as long as we mount the package files in the connectors setup.py